create_ancient.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
It is just a quick throw together script to both create a world file and draw the ancient map.I think that longer term it goes away. Some how, the command line use of generator.py should allow complete creation of the world file and drawing the ancient map.